templates/includes/card-group-record-summary-no-image.html

```diff
@@ -15,7 +15,8 @@
                     {{ record.reference_number }} -
                 </span>
                 {% endif %}
-                {% firstof description_override|safe record.summary_title|striptags %}
+                {# default_if_none for those template calls that do not share description_override #}
+                {% firstof description_override|default_if_none:""|safe record.summary_title|striptags %}
```

Comment:
Question for @William-Blackie:
I'm unsure whether the `|safe` filter is needed here. If `description_override` is RichText, that should already be marked as 'safe' after applying the `|richtext` filter. Do we know what other types of value we are expecting here?